ONSdigital/rm-collection-exercise-service
# RAS-1310 Upgrade liquibase dependency in collection exercise service

# What and why?
Upgrades liquibase to the most up to date version and moves commons io from a test only dependency to being including when compiling which is what caused liqubase to break
# How to test?
- run acceptance tests in your environment
- deploy this pr to your environment
- make sure that it deploys correctly and you can access collection exercises in rOps
- run the acceptance tests again and/or try to access collection exercises through rOps/frontstage
# Jira
[RAS-1310](https://jira.ons.gov.uk/browse/RAS-1310)
